tests/unittests/test_dev_mode.py

```diff
@@ -0,0 +1,64 @@
+import json
+from requests.exceptions import HTTPError
+
+import unittest
+from unittest.mock import patch, MagicMock
+
+from tap_adroll.client import AdrollClient
+
+
+class Test_ClientDevMode(unittest.TestCase):
+    """Test the dev mode functionality."""
+ 
+    # Data to be written
+    mock_config = {
+    "access_token": "sample_access_token",
+    "refresh_token": "sample_refresh_token",
+    "client_id": "sample_client_id",
+    "client_secret": "sample_client_secret"
+    }
+    tmp_config_filename = "adroll_config.json"
+    # Serializing json
+    json_object = json.dumps(mock_config, indent=4)
+    
+    # Writing to sample.json
+    with open(tmp_config_filename, "w") as outfile:
+        outfile.write(json_object)
```

Comment:
I will strongly suggest deleting the temp config once the unit test is executed. If we don't delete it, I think it will keep on accumulating the temp configs (as the everyday job runs) within circleci server.

Reply:
made the changes